src/main/java/teammates/common/datatransfer/attributes/StudentProfileAttributes.java

```diff
@@ -192,8 +214,10 @@ public Builder withNationality(String nationality) {
             return this;
         }
 
-        public Builder withGender(String gender) {
-            profileAttributes.gender = isGenderValid(gender) ? gender : "other";
+        public Builder withGender(Gender gender) {
```

Comment:
@xpdavid Would it be better to take a `String` as parameter here instead and then do`profileAttributes.gender = getGenderEnumValue(gender)` ?